dev-workflow-ui/src/ch/ivyteam/workflowui/cms/IvyBean.java

```diff
@@ -0,0 +1,35 @@
+package ch.ivyteam.workflowui.cms;
+
+import java.nio.file.Path;
+
+import javax.faces.bean.ApplicationScoped;
+import javax.faces.bean.ManagedBean;
+
+import ch.ivyteam.ivy.cm.exec.ContentManagement;
+import ch.ivyteam.ivy.cm.exec.ContentManagementManager;
+import ch.ivyteam.ivy.environment.Ivy;
+import ch.ivyteam.workflowui.util.UrlUtil;
+
+@ManagedBean(name = "ivy")
+@ApplicationScoped
+public class IvyBean {
```

Comment:
this bean looks already pretty. I think I can implement this fallback directly on IvyJsf bean directly. right?